public/app/features/transformers/editors/CalculateFieldTransformerEditor.tsx

```diff
@@ -250,6 +266,155 @@ export class CalculateFieldTransformerEditor extends React.PureComponent<
     );
   }
 
+  //---------------------------------------------------------
+  // Window functions
+  //---------------------------------------------------------
+
+  updateWindowOptions = (v: WindowOptions) => {
+    const { options, onChange } = this.props;
+    console.log(v);
```

Comment:
does this need to be logged?

Reply:
Whoops, I was sure I searched the diff for console log 🤦‍♂️